README.md

```diff
@@ -85,7 +85,8 @@ See `dotnet-gitchanges\appsettings.json` for the default settings file. Any sett
       "OverrideSource": "someOverrideSource.txt"
     },
     "FileSource": "someHistorialChanges.txt",
-    "MultiProject": false
+    "MultiProject": false,
+    "UseGitTags": false

```

Comment:
This still says UseGitTags instead of VersionFromGitTags?

Reply:
This is really strange, it is changed on the branch, but still shows the old  value here.